test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/controllers/ProactiveChatController.spec.js

```diff
@@ -19,28 +43,7 @@ describe("ProactiveChatController", () => {
         commonChatController.nuanceFrameworkLoaded(window);
         proactiveChatController.launchProactiveChat();
 
-        expect(sdk.getOpenerScripts).toHaveBeenCalled();
-        expect(sdk.chatDisplayed).toHaveBeenCalled();
-    })
-
-    it("appends chat transcript div to page when no div id is found on page", () => {
-        const commonChatController = new CommonChatController();
-
-        commonChatController._showChat();
-
-        expect(document.getElementById("ciapiSkinChatTranscript").innerHTML).not.toBe(null);
-
-    });
-
-    it("appends chat transcript div to page when some div id is found on page", () => {
-        const commonChatController = new CommonChatController();
-        let chatContainer = document.createElement("div");
-        chatContainer.setAttribute("id", "HMRC_CIAPI_Fixed_1");
-        document.body.appendChild(chatContainer);
-
-        commonChatController._showChat();
-
-        expect(document.getElementById("ciapiSkinChatTranscript").innerHTML).not.toBe(null);
```

Comment:
has this really all been removed or are these tests still in place?

Reply:
Added back in, well spotted